repo2docker/buildpacks/conda/install-base-env.bash

```diff
@@ -36,7 +38,14 @@ update_dependencies: false
 # channel_priority: flexible
 EOT
 
-micromamba install conda=${CONDA_VERSION} mamba=${MAMBA_VERSION} -y
+# TODO: remove channels and conda-libmamba-solver once 2.0.0 is used.
+micromamba install \
+    -c conda-canary/label/conda-libmamba-solver-pr-457 \
+    -c conda-forge/label/mamba_dev \
+    -c conda-forge \
+    "conda-libmamba-solver>=${CONDA_VERSION}" \
```

Comment:
How can we make sure this is taking the right dev version and not `24.7.0`?

Reply:
Channel priority. We can use the fully qualified spec otherwise, i.e.:

```suggestion
    "conda-canary/label/conda-libmamba-solver-pr-457::conda-libmamba-solver>=${CONDA_VERSION}" \
```